hpa/locust/kustomization.yaml

```diff
@@ -4,7 +4,7 @@ kind: Kustomization
 namespace: recipe
 
 resources:
-  - github.com/thestormforge/examples/voting-webapp/application
+  - github.com/thestormforge/examples/webserver/voting-webapp/application
```

Comment:
Actually, we decided to use relative paths as a convention when in the same repo, unless there is reason to fully qualify in this case?